accel_code/excel_lookup.py

```diff
@@ -13,8 +14,8 @@ def excel_lookup(lab_id, date, excel_file):
     
     # convert all ID's to strings
     df.LabID = df.LabID.astype(str)
-    sess = df[( df['LabID'].str.contains(sub_id) ) & ( df['File date'] == date )]
-    sess_id = str(sess['LabID']).split('_')[1]
-    project = str(sess['Project'])
+    sess = df[( df['LabID'].str.contains(lab_id) ) & ( df['File date'] == date )]
```

Comment:
we can place a check here to see if the dataframe is more than one row.

```
if len(ses.index) > 1:
    raise IndexError("{lab_id}: {date} has multiple entries in excel file".format(lab_id=lab_id, date=date)
```
and you can raise a similar error message if the dataframe is empty.